crates/polars-arrow/src/legacy/kernels/binary.rs

```diff
@@ -0,0 +1,10 @@
+use crate::array::{Array, ArrayRef, BinaryViewArray, UInt32Array};
+use crate::buffer::Buffer;
+use crate::datatypes::ArrowDataType;
+
+pub fn binary_size_bytes(array: &BinaryViewArray) -> ArrayRef {
```

Comment:
Can you move this to `polars-compute`?

Reply:
I was wondering about the "legacy" subdirectory ... ;)

Reply:
Done; let me know if you prefer it in a different file/subdir.